gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/JobStatistics.java

```diff
@@ -61,7 +61,7 @@ private ExtractStatistics(Builder builder) {
     /**
      * Returns the number of files per destination URI or URI pattern specified in the extract job.
      * These values will be in the same order as the URIs specified by
-     * {@link ExtractJobInfo#destinationUris()}.
+     * {@link ExtractJobConfiguration#destinationUris()}.
```

Comment:
I think we discussed this elsewhere. Removing the `@SuppressWarnings("unchecked")` causes the warning:

> fromPb(com.google.api.services.bigquery.model.JobStatistics) in com.google.gcloud.bigquery.JobStatistics.ExtractStatistics overrides <T>fromPb(com.google.api.services.bigquery.model.JobStatistics) in com.google.gcloud.bigquery.JobStatistics
  return type requires unchecked conversion from com.google.gcloud.bigquery.JobStatistics.ExtractStatistics to T